Content/.vscode/launch.json

```diff
@@ -1,51 +0,0 @@
-{
```

Comment:
why did we decide to remove launch and tasks files for vscode? do we plan to describe it in docs?

Reply:
I'm happy to put it back, but it added further complexity to the FAKE script and template.

Reply:
We can, again, solve this with docs and samples.

Reply:
Ok let's do it with docs - I'll edit your list directly in https://github.com/SAFE-Stack/docs/issues/154

Reply:
Ok I can see it's already there as "How do I debug my SAFE apps?"